src/translations/dictionary.ts

```diff
@@ -14,6 +14,7 @@ const dictionary = {
       `Aktiver for reisesøk. Aktiver to ganger for å nullstille søket.`,
       'Activate for search assistant. Activate twice to reset search.',
     ),
+    map: _(`Kart`, 'Map'),
```

Comment:
would move these translations to a separate relevant named file in a separate PR